integration-test/src/main/java/com/sequenceiq/it/cloudbreak/VmTypeTests.java

```diff
@@ -0,0 +1,217 @@
+package com.sequenceiq.it.cloudbreak;
```

Comment:
Could you please turn on Idea inspections and fix all possible code smells?

Reply:
Thank you for your hint on this. This has been turned on and regularly used.

Reply:
(y)